backend/app/services/adapters/task_kinds/queries.py

```
@@ -1131,3 +1131,129 @@ def get_task_skills(
+
+        # Use user_subtask as the primary subtask for execution
+        subtask = user_subtask
+
+        # Get team
+        from app.schemas.kind import Task as TaskCRD
+
+        task_crd = TaskCRD.model_validate(task.json)
+        if not task_crd.spec.teamRef:
+            raise ValueError(f"Task {task_id} has no team reference")
+
+        from app.models.kind import Kind
```

Ruff (0.15.2)

[error] Redefinition of unused `Kind` from line 19: `Kind` redefined here

Remove definition: `Kind`

(F811)